src/ui/ManageCoursesUi.csproj

```diff
@@ -28,7 +28,7 @@
   <Import Condition="Exists('$(ProjectDir)dev-sc-shared.targets')" Project="$(ProjectDir)dev-sc-shared.targets" />
 
   <ItemGroup Condition="Exists('$(ProjectDir)dev-mc-api.targets')==false">
-    <PackageReference Include="GovUk.Education.ManageCourses.ApiClient" Version="0.22.0.*" />
+    <PackageReference Include="GovUk.Education.ManageCourses.ApiClient" Version="0.22.0-death-to-swagger-generation.31" />
```

Comment:
don't forget to update this once the api is merged

Reply:
When you approve it then this can be updated.

So you better work faster!

Reply:
done